astropy/table/table.py

```diff
@@ -93,6 +93,8 @@ class used to create new non-mixin columns, and this is a function of
     "Table.convert_unicode_to_bytestring",
 ]
 
+__doctest_requires__ = {("Table.from_pandas", "Table.to_pandas"): ["pandas"]}
```

Comment:
I also found this by accident because doctest was giving me ImportError locally when I didn't have pandas installed.